eng/versioning/version_client.txt

```diff
@@ -34,6 +34,7 @@ com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-eventhubs;5.1.2;5.2.0-beta.3
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.1.2;1.2.0-beta.3
 com.azure:azure-messaging-servicebus;7.0.0-beta.5;7.0.0-beta.6
+com.azure:azure-messaging-servicebus-perf;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
Missing the track 1 performance test library.

Reply:
changed it